2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -425,9 +425,10 @@ Nel quinto sprint il team svolge compiti correttivi incentrati sul miglioramento
 
 - *Responsabile*: al fine di coordinare le attività e contattare l'azienda Proponente;
 - *Amministratore*: al fine di gestire gli strumenti GitHub e Jira;
-- *Verificatore*: al fine di verificare la correttezza delle modifiche ai documenti e di effettuare un controllo sulla validità e formulazione degli use case individuati e del documento di Analisi dei Requisiti;
 - *Analista*: al fine di redarre il documento di Analisi dei Requisiti;
-- *Progettista*: al fine di sviluppare i POC.
+- *Progettista*: al fine di continuare lo studio sulla libreria Three.js;
+- *Programmatore*: al fine di sviluppare i POC relativi allo studio della libreria;
```

Comment:
in glossario, c'è un errore sulla voce relativa il POC, uno spazio bianco in più ("POC " anzichè "POC"), che fa visualizzare male il testo, puoi correggerlo per favore?